Lib/uuid.py

```diff
@@ -342,11 +342,29 @@ def _popen(command, *args):
                             env=env)
     return proc
 
+# For MAC (a.k.a. IEEE 802, or EUI-48) addresses, the second least significant
+# bit of the first octet signifies whether the MAC address is universally (0)
+# or locally (1) administered.  Network cards from hardware manufacturers will
+# always be universally administered to guarantee global uniqueness of the MAC
+# address, but any particular machine may have other interfaces which are
+# locally administered.  An example of the latter is the bridge interface to
+# the Touch Bar on MacBook Pros.
+#
+# This bit works out to be the 42nd bit counting from 1 being the least
+# significant, or 1<<41.  We'll skip over any locally administered MAC
+# addresses, as it makes no sense to use those in UUID calculation.
```

Comment:
Instead of "it makes no sense", I suggest to explain it differently, something like: "prefer universally administered addresses since UUID1 objects should be unique".